draft-ietf-rtcweb-jsep.xml

```diff
@@ -1102,7 +1102,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         bundle-only. This policy aims to minimize
                         candidate gathering and maximize multiplexing,
                         at the cost of less compatibility with legacy
-                        endpoints.</t>
+                        endpoints. When acting as answerer, the browser
```

Comment:
I just copied the terminology used in the RTCP mux section. I'll change that as well.